drivers/sensor/isl29125/Kconfig

```diff
@@ -1,14 +1,52 @@
-# ISL29125 Color Light to Digital Converter configuration options
+#ISL29125 Color Light to Digital Converter configuration options
 
-# Copyright (c) 2016 Intive
-# SPDX-License-Identifier: Apache-2.0
+#Copyright(c) 2016 Intive
+#SPDX - License - Identifier : Apache - 2.0
 
 menuconfig ISL29125
-	bool "ISL29125 Color Light to Digital Converter"
-	depends on I2C
-	help
-	  Enable driver for ISL29125 Color Light to Digital Converter.
+        bool "ISL29125 Color Light to Digital Converter"
+        depends on I2C
+        help
+                Enable driver for ISL29125 Color Light to Digital Converter.
 
 if ISL29125
 
+choice
+        prompt "Trigger mode"
+        depends on GPIO
+        default ISL29125_TRIGGER_NONE
+        help
+                Specify the type of triggering to be used by the driver.
```

Comment:
These lines should all start with one tab. The "Specify..." line should have two additional spaces after the tab. Please make similar changes throughout this file.